src/manifest.json

```diff
@@ -28,6 +28,12 @@
     "scripts": ["background.js"],
     "persistent": true
   },
+  "browser_specific_settings": {
```

Comment:
Can we remove `browser_specific_settings` (lines 31-36)? Mozilla will complain if we introduce that id.

Reply:
I added those `browser_specific_settings` because without them Firefox refused to add the extension. Maybe in the regular deployment process it won’t be necessary but I’m not sure.
Anyway I’m not sure why adding an ID could be a problem, it’s a pretty common practice.